src/math/curve.js

```diff
@@ -20,7 +20,8 @@ import { CurveEvaluator } from './curve-evaluator.js';
  * * {@link pc.CURVE_STEP}
  *
  * Defaults to {@link pc.CURVE_SMOOTHSTEP}.
- *
+ * @property {number} tension The tension used when type is CURVE_SPLINE.
```

Comment:
What is tension?

Reply:
Lol. I wondered whether to link to the wikipedia entry? https://en.wikipedia.org/wiki/Cubic_Hermite_spline#Cardinal_spline

Reply:
That's fine to do that if you think the developer would find it informative. I normally find a lot of Wikipedia math explanations a bit overcomplicated, so a nice easy-to-understand summary of the meaning can be nice to have in the docs. Up to you.

Reply:
Having it clear that it should be between 0 and 1 and a value of 1 means a straight line graph (does it? I'm really not sure 😅 )

Reply:
done